src/huggingface_hub/hf_file_system.py

```diff
@@ -27,7 +27,7 @@
 # Regex used to match special revisions with "/" in them (see #1710)
 SPECIAL_REFS_REVISION_REGEX = re.compile(
     r"""
-    (^refs\/convert\/parquet) # `refs/convert/parquet` revisions
+    (^refs\/convert\/\w+)     # `refs/convert/parquet` revisions
```

Comment:
just in case we add other convert branches 🙃 

(we're currently discussing moving the duckdb indexes - could be in the parquet branch but named differently or in another convert/ branch)